### PR TITLE
Update sig docs generator to support chairs/tech leads

### PR DESCRIPTION
**Pull request by cblecker**
Per https://git.k8s.io/community/committee-steering/governance/sig-governance-template-short.md, the role of "SIG Lead" is becoming "Chair" and a new role of "Technical Lead" is being created.

This modifies the generator to support these new roles. Note, I've only regenerated sig-contributor-experience as an example for ease of review.

### CODE REVIEW

File: generator/app.go [hunk 1]
```diff
@@ -85,22 +84,28 @@ type GithubTeams struct {
 	Description string
 }
 
-// Subproject represenst a specific subproject owned by the group
+// Subproject represensts a specific subproject owned by the group
```
Comment (jamiehannaford) [inline generator/app.go]: typo here

File: generator/sig_readme.tmpl [hunk 2]
```diff
@@ -13,18 +13,35 @@
...
-## Leads
-{{- range .Leads }}
+{{- end }}
+{{- if .Leadership }}
+
+## Leadership
+{{- if .Leadership.Chairs }}
+
+### Chairs
+The Chairs of the SIG run operations and processes governing the SIG.
```
Comment (jamiehannaford) [inline generator/sig_readme.tmpl]: maybe replace "processes governing" with just "govern"?

File: generator/app.go (hunk 1 above)
Comment (cblecker) [reply generator/app.go]: Fixed!

File: generator/sig_readme.tmpl (hunk 2 above)
Comment (cblecker) [reply generator/sig_readme.tmpl]: I don't disagree, but wanted to use the similar wording to the short template the steering committee provided here: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance-template-short.md#roles

Comment (idvoretskyi) [reply generator/sig_readme.tmpl]: I'd prefer keeping the @cblecker's wording.

**Comment by cblecker** [comment]
@idvoretskyi the failure is expected, because I haven’t regenerated all the different sigs yet (makes it easier to review). Once I get a thumbs up, I’ll regenerate the rest.

**Comment by grodrigues3** [comment]
/lgtm
/hold 

feel free to merge after regenerating

**Comment by cblecker** [comment]
/hold cancel

@grodrigues3 mind re-LGTMing? I've regenerated the rest of the docs

**Pull request merged**